src/universal/utils/makeHref.js

```diff
@@ -1,7 +1,9 @@
-export default function makeMeetingUrl(relativeLink) {
+import makeSSRAppLink from 'server/utils/makeAppLink';
+
+export default function makeHref(relativeLink) {
   if (typeof window !== 'undefined') {
     return `${window.location.origin}${relativeLink}`;
   }
-  // default for SSR/error catching
-  return 'https://www.parabol.co';
+  // default for SSR
+  return makeSSRAppLink(relativeLink);
```

Comment:
oh fo sho! it'll make our test coverage go down since it shouldn't ever get called, but we ain't got no test coverage.